oracle/config/websocket.go

```diff
@@ -42,6 +42,10 @@ const (
 	// DefaultPingInterval is the default interval at which the provider should send
 	// ping messages to the server.
 	DefaultPingInterval = 0 * time.Second
+
+	// DefaultMaxSubscriptionsPerConnection is the default maximum subscriptions
+	// an provider can handle per-connection.
```

Comment:
nit: can you explain what happens when this value is 0? Does it just establish a single connection?